doc/classes/RayCast2D.xml

```diff
@@ -56,6 +56,21 @@
 			<return type="int" />
 			<description>
 				Returns the shape ID of the first object that the ray intersects, or [code]0[/code] if no object is intersecting the ray (i.e. [method is_colliding] returns [code]false[/code]).
+				To get the intersected shape node, use:
+				[codeblocks]
+				[gdscript]
+				var target = get_collider() # A CollisionObject2D.
```

Comment:
Could you also not fix the example, instead of adding a note for the exception to the rule?
```suggestion
				var target = get_collider() # A CollisionObject2D or TileMap.
```